dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/data-storage.test.js

```diff
@@ -22,7 +25,253 @@ define([
+                expect(model.initConfig()).toBeDefined();
+            });
+
+            it('Check returned value type if method called without arguments', function () {
+                var type = typeof model.initConfig();
+
+                expect(type).toEqual('object');
+            });
+
+            it('Check this.dataScope property (is modify in initConfig method)', function () {
+                model.dataScope = null;
+                model.initConfig();
+                expect(typeof model.dataScope).toEqual('object');
+            });
+
+            it('Check this._requests property (is modify in initConfig method)', function () {
+                model._requests = null;
+                model.initConfig();
+                expect(typeof model._requests).toEqual('object');
+            });
+        });
+
+        describe('"getByIds" method', function () {
+
+            var model = new DataStorage({
+                dataScope: 'magento'
+            });
+
+            it('check for defined', function () {
+                expect(model.hasOwnProperty('getByIds')).toBeDefined();
+            });
+
+            it('check method type', function () {
+                expect(typeof model.getByIds).toEqual('function');
+            });
+
+            it('Check returned value if method called with argument', function () {
+                var ids = [1,2,3];
+
+                expect(model.getByIds(ids)).toBeDefined();
+            });
+
+            it('check returned type if method called with argument', function () {
+                var ids = [1,2,3],
+                    type = typeof model.getByIds(ids);
+
+                expect(type).toEqual('boolean');
+            });
+
+            it('Return false if "getByIds" has been called', function () {
+                var ids = [1,2,3];
+
+                expect(model.getByIds(ids)).toEqual(false);
+            });
+
+            it('Return array if "getByIds" has been called', function () {
+                var ids = [1];
+
+                model = new DataStorage({
+                    dataScope: 'magento',
+                    data: {
+                        1: {
+                            id_field_name: 'entity_id',
+                            entity_id: '1'
+                        }
+                    }
+                });
+
+                expect(typeof model.getByIds(ids)).toEqual('object');
+            });
+
+        });
+
+        describe('"getIds" method', function () {
+
+            var model = new DataStorage({
+                dataScope: 'magento'
+            });
+
+            it('check for defined', function () {
+                expect(model.hasOwnProperty('getIds')).toBeDefined();
+            });
+
+            it('check method type', function () {
+                expect(typeof model.getIds).toEqual('function');
+            });
+
+            it('check returned value if method called with argument', function () {
```

Comment:
I don't think it's beneficial to test booth return type and return value - only one check for return value would be enough.

It'd be better to check the case when no arguments are passed to `getIds` method instead.